docs/mp/restclient/09_rest-client.adoc

```diff
@@ -113,4 +113,4 @@ Then create runnable method the same way as it is described in
 `Creating new client - Interface`,but with baseUri `http://localhost:8080/greet`
 and the above interface.
 
-By calling `GreetRestClient.getDefaultMessage()` you reach the endpoint of Helidon quickstart.
+By calling `GreetRestClient.getDefaultMessage()` you reach the endpoint of Helidon quickstart.
```

Comment:
This looks like a white space change. This should be displayed in the diff as "File renamed without changes"